compiler/rustc_builtin_macros/src/autodiff.rs

```diff
@@ -431,6 +428,54 @@ mod llvm_enzyme {
         }
         body.stmts.push(ecx.stmt_semi(black_box_remaining_args));
 
+        (body, primal_call, black_box_primal_call, blackbox_call_expr)
+    }
+
+    /// We only want this function to type-check, since we will replace the body
+    /// later on llvm level. Using `loop {}` does not cover all return types anymore,
+    /// so instead we build something that should pass. We also add a inline_asm
+    /// line, as one more barrier for rustc to prevent inlining of this function.
+    /// FIXME(ZuseZ4): We still have cases of incorrect inlining across modules, see
+    /// <https://github.com/EnzymeAD/rust/issues/173>, so this isn't sufficient.
+    /// It also triggers an Enzyme crash if we due to a bug ever try to differentiate
```

Comment:
Missing words, preexisting, but...

Reply:
I removed the FIXME since we figured out that it was due to the non-fat-lto usage.
I also reworded it a bit, but I am not 100% sure which words you are missing, so please feel free to suggest something else.